### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by jonathanraes**
Refactored app, fixed a lot of checkstyle and mapped S to opening the cameraframe (removed the standard opening of the frame)
What needs to be done:
Testing
remove hardcoding of obstacle sizes
fixing additional tool errors

### CODE REVIEW

File: src/main/java/com/funkydonkies/camdetect/Mat2Image.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
     private byte[] dat;
     private Mat bg = new Mat();
     private Mat res = new Mat();
-    private Boolean bgSet = false;
+    private static Boolean bgSet = false;
```
Comment (Danilodum) [inline src/main/java/com/funkydonkies/camdetect/Mat2Image.java:29]: need better name

File: src/main/java/com/funkydonkies/controllers/SplineCurveController.java [hunk 2]
```diff
@@ -2,78 +2,94 @@
...
 	private boolean updateEnabled = false;
-	
-	public SplineCurveController(final Bridge b, final SplineCurve sp) {
-		this.splineCurve = sp;
-		this.bridge = b;
-	}
-	
+
+	private static RigidBodyControl oldRigi;
+	private static RigidBodyControl rigi;
```
Comment (Danilodum) [inline src/main/java/com/funkydonkies/controllers/SplineCurveController.java:50]: collision needs better implementation than removing and creating rigidbodies

**Comment by Danilodum** [comment]
good refactor, no tool errors

**Comment by ghost** [comment]
Looks good and clean

**Pull request merged**